sdk/helper/certutil/helpers.go

```diff
@@ -312,8 +312,12 @@ func ParsePEMBundle(pemBundle string) (*ParsedCertBundle, error) {
 		}
 	}
 
-	if err := parsedBundle.Verify(); err != nil {
-		return nil, errutil.UserError{Err: fmt.Sprintf("verification of parsed bundle failed: %s", err)}
+	if certPath != nil && len(certPath) > 1 {
```

Comment:
Please consider simplifying this statement to:
```
if len(certPath) > 1 {
```

Reply:
Neat!  fixed.